deploy/azuredeploy.json

```diff
@@ -84,6 +89,17 @@
       },
       "properties": {}
     },
+    {
+      "type": "Microsoft.DocumentDB/databaseAccounts",
+      "apiVersion": "2021-10-15",
```

Comment:
Are we intentionally entered the fixed older version on here?

Reply:
No, that was an oversight. Updated to latest api version for all the newly added resources.